cartopy/meta.yaml

```diff
@@ -39,8 +38,7 @@ requirements:
     - owslib
     - numpy
     - proj4
-    - shapely ==1.4.3.issue177  # [osx]
-    - shapely                   # [not osx]
+    - shapely ==1.5.6
```

Comment:
There are good reasons for the pinned version of shapely on OSX:

https://github.com/Toblerity/Shapely/issues/177